consensus/istanbul/core/handler.go

```diff
@@ -193,7 +199,26 @@ func (c *core) handleCheckedMsg(msg *message, src istanbul.Validator) error {
 	return errInvalidMessage
 }
 
-func (c *core) handleTimeoutMsg() {
+func (c *core) handleTimeoutMsg(view *istanbul.View) {
+	// TODO-Klaytn-Istanbul: Check whether this statement is required. EN/PN should not handle consensus msgs.
+	if c.backend.NodeType() != common.CONSENSUSNODE {
+		logger.Warn("Received an istanbul timeout message from EN or PN",
```

Comment:
This is the right log if I understand right?
```suggestion
		logger.Warn("This is not a consensus node, but received an istanbul timeout message",
```

Reply:
Thanks. I changed it